arch/arm/core/cortex_m/tz/CMakeLists.txt

```diff
@@ -18,6 +18,21 @@ if(CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS)
   zephyr_ld_options(
     ${LINKERFLAGPREFIX},--cmse-implib
   )
+
+  if(CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS)
```

Comment:
@SebastianBoe could you help-out here? Is this the right way to do it?
@oyvindronningstad I am now a bit confused about what you want to achieve; isn't this fulfilled by the above ld-option? Pls, elaborate.

Reply:
Went through it with @SebastianBoe now, and made a few changes. BTW, something like this is needed for multi-image builds, to instruct Ninja to link SPM before the app, so the file exists when needed.